cmd/meroxa/root/apps/deploy.go

```diff
@@ -155,6 +215,8 @@ func (d *Deploy) Execute(ctx context.Context) error {
 		return err
 	}
 
+	d.gitWarnings(ctx)
```

Comment:
We should interrupt the deploying process if there are changes untracked. Maybe, if `gitWarnings` returned an error, we could do that.

Reply:
so it's not so much a warning as an error? that does make sense because it is the main branch.

Reply:
@janelletavares that's correct.